sdk/identity/identity/review/identity.api.md

```diff
@@ -288,10 +296,15 @@ export interface ManagedIdentityCredentialResourceIdOptions extends TokenCredent
     resourceId: string;
 }
 
+// @public
+export interface MultiTenantTokenCredentialOptions extends TokenCredentialOptions {
```

Comment:
Will there be more properties in the future for `MultiTenantTokenCredentialOptions`? I wonder whether adding this indirection benefits end users if it's just one property.

Reply:
I think in the long run it's worth it not to have code duplication across 15 other options types here where if there are additional tweaks we need to make for multitenant auth, we can do it in one place.

Reply:
Totally agree that it improves our code quality! Just saying that we favor customers benefit over our owns and I didn't see other languages introducing this.

Reply:
Right.  Others have reviewed from other identity teams and saw no issue, so I think we're good on this.